python/demos/character_commands.py

```diff
@@ -10,16 +10,25 @@ class EnableAgentDemo(Demo):
     def __init__(self):
         super().__init__(
             name="Enable Agent",
-            summary="Enable or disable an agent after the simulation has started. An enabled agent uses the python bridge to generate its actions.",
+            summary="Enable or disable an agent",
             category=CATEGORY,
             function=self.enable_agent_demo,
         )
 
     def enable_agent_demo(self, bridge: RuntimeBridge):
         """
-        Enable or disable an agent
-
-        :param bridge: The bridge to the runtime.
+        On simulation start, the chosen character's bridge agent is enabled or disabled. Enabled agents use the
+        Bridge to generate their actions and will contribute to the overall cost of running the simulation. Disabled
```

Comment:
.. will contribute to the overall cost of running the simulation [when using a paid LLM service like OpenAI.]